codegenerator/cli/src/config_parsing/chain_helpers.rs

```diff
@@ -32,227 +32,304 @@ use crate::constants::DEFAULT_CONFIRMED_BLOCK_THRESHOLD;
 #[strum(serialize_all = "kebab-case")]
 #[repr(u64)]
 pub enum Network {
-    #[subenum(
-        HypersyncNetwork,
-        NetworkWithExplorer,
-        GraphNetwork(serde(rename = "mainnet"))
-    )]
-    EthereumMainnet = 1,
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    Amoy = 80002,
+
+    #[subenum(NetworkWithExplorer, GraphNetwork)]
+    ArbitrumGoerli = 421613,
+
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    ArbitrumNova = 42170,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
-    Goerli = 5,
+    ArbitrumOne = 42161,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
-    Optimism = 10,
+    ArbitrumSepolia = 421614,
+
+    #[subenum(GraphNetwork)]
+    AuroraTestnet = 1313161555,
+
+    #[subenum(HypersyncNetwork, GraphNetwork, NetworkWithExplorer)]
+    Avalanche = 43114,
+
+    #[subenum(HypersyncNetwork, GraphNetwork)]
+    // Blockscout: https://explorer.aurora.dev/
+    Aurora = 1313161554,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
     Base = 8453,
+
+    #[subenum(GraphNetwork(serde(rename = "base-testnet")))]
+    BaseGoerli = 84531,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer)]
     // explorers:
     // https://sepolia.basescan.org/
     // https://base-sepolia.blockscout.com/
     BaseSepolia = 84532,
+
+    #[subenum(HypersyncNetwork)]
+    BerachainBartio = 80084,
+
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    Blast = 81457,
+
+    #[subenum(HypersyncNetwork, NetworkWithExplorer)]
+    BlastSepolia = 168587773,
+
+    #[subenum(HypersyncNetwork)]
+    // explorers:
+    // https://bobascan.com/ (not etherscan)
+    Boba = 288,
+
     #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
     Bsc = 56,
-    #[subenum(GraphNetwork)]
-    PoaSokol = 77,
-    #[subenum(GraphNetwork)]
-    Mumbai = 80001,
+
     #[subenum(HypersyncNetwork, GraphNetwork(serde(rename = "chapel")))]
     BscTestnet = 97,
+
+    #[subenum(HypersyncNetwork, GraphNetwork, NetworkWithExplorer)]
+    Celo = 42220,
+
     #[subenum(GraphNetwork)]
-    PoaCore = 99,
-    #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
-    Gnosis = 100,
+    CeloAlfajores = 44787,
+
+    #[subenum(HypersyncNetwork)]
+    Chiliz = 8888,
+
     #[subenum(GraphNetwork)]
-    Fuse = 122,
-    #[subenum(HypersyncNetwork, NetworkWithExplorer, GraphNetwork)]
-    Fantom = 250,
+    Clover = 1023,
+
+    #[subenum(HypersyncNetwork)]
+    // Explorers:
+    // https://crab.subscan.io/
+    Crab = 44,
+
+    #[subenum(HypersyncNetwork)]
+    // Explorers:
+    // Blockscout: https://explorer-mainnet-cardano-evm.c1.milkomeda.com/
+    C1Milkomeda = 2001,
+
+    #[subenum(HypersyncNetwork)]
+    // blocksout: https://explorer.devnet.citrea.xyz/
+    CitreaDevnet = 62298,
```

Comment:
This is the only addition. It would be nice to have in a release for my workshop tomorrow but I can also work around it for now.